yew-macro/tests/html_macro/html-if-fail.stderr

```diff
@@ -0,0 +1,37 @@
+error: unexpected end of input, expected curly braces
+ --> $DIR/html-if-fail.rs:4:5
+  |
+4 |     html! { if {} };
+  |     ^^^^^^^^^^^^^^^^
+  |
+  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
+
+error: unexpected end of input, expected `if`
+ --> $DIR/html-if-fail.rs:6:5
+  |
+6 |     html! { if true {} else };
+  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^
+  |
+  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
+
+error: unexpected end of input, expected curly braces
+ --> $DIR/html-if-fail.rs:7:5
+  |
+7 |     html! { if true {} else if {} };
+  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  |
+  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
+
+error: unexpected end of input, expected `if`
+ --> $DIR/html-if-fail.rs:8:5
+  |
+8 |     html! { if true {} else if true {} else };
+  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I did not do at all what you said because it didn't work but I manage to make those error messages and span :ok_hand: You probably should check the english though